src/app/pages/search/search-no-result/search-no-result.component.html

```diff
@@ -20,8 +20,7 @@ <h1 class="h2">{{ 'search.noResult.heading' | translate }}</h1>
     <ish-search-box
       [configuration]="{
         placeholder: 'search.searchbox.instructional_text' | translate,
-        autoSuggest: true,
-        maxAutoSuggests: 10
+        autoSuggest: false
```

Comment:
If `autoSuggest` is enabled for the search box in no results page, the search suggest no longer works, the styling is completly broken. I assume there is no reason why suggest should not be possible on the no search results page even though it was turned off by default. Is this on purpose compared to the IAD we had before with enabled suggests for no search results?

![grafik](https://github.com/user-attachments/assets/74e6e70a-cdc0-40f2-a954-2e63b977f345)

Reply:
The same goes for the search box integration on the error page. It is not styled either with `autoSuggest: true`.
On the error page it is working. On the no search results page it seems impossible to get into the input to enter a new search term.

Reply:
I'll remove the autosuggest parameter for the _ish-search-box_ configuration of the _search-no-result_ component. UI behavior and look has to defined first. Is part of subsequent feature development.